modules/howtos/examples/KvOperations.scala

```diff
@@ -24,368 +24,428 @@ import com.couchbase.client.scala.kv.{GetOptions, InsertOptions, MutationResult,
 
 import scala.concurrent.duration._
 import scala.util.{Failure, Success, Try}
-// #end::imports[]
+// end::imports[]
 
-// #tag::cc-create[]
+// tag::cc-create[]
 case class Address(line1: String)
 case class User(name: String, age: Int, addresses: Seq[Address])
-// #end::cc-create[]
+// end::cc-create[]
 
-// #tag::cc-codec[]
+// tag::cc-codec[]
 object User {
   implicit val codec: Codec[User] = Codec.codec[User]
 }
-// #end::cc-codec[]
+// end::cc-codec[]
 
 object KvOperations {
   def main(args: Array[String]) {
 
-    def json1() {
-
-      // User("Eric Wimp", 9, Seq(Address("29 Acacia Road")))
-// #tag::json1[]
-      val json = JsonObject(
-        "name" -> "Eric Wimp",
-        "age" -> 9,
-        "addresses" -> JsonArray(JsonObject("address" -> "29 Acacia Road"))
-      )
-
-      val str = json.toString()
-// """{"name":"Eric Wimp","age":9,"addresses":[{"address","29 Acacia Road"}]}"""
-// #end::json1[]
-
-// #tag::json-mutable[]
-      val obj = JsonObject.create.put("name", "Eric Wimp")
-      obj.put("age", 9)
-      val arr = JsonArray.create
-      arr.add(JsonObject("address" -> "29 Acacia Road"))
-      obj.put("addresses", arr)
-// #end::json-mutable[]
-
-// #tag::json2[]
-      json.str("name") // "Eric Wimp"
-      json.arr("addresses").obj(0).str("address") // "29 Acacia Road"
-// #end::json2[]
-
-      // #tag::json3[]
-      json.dyn.name.str // "Eric Wimp"
-      json.dyn.addresses(0).address.str // "29 Acacia Road"
-// #end::json3[]
-
-// #tag::json4[]
-      val safe: JsonObjectSafe = json.safe
-
-      val r: Try[String] = safe.str("name")
-
-      r match {
-        case Success(name) => println(s"Their name is $name")
-        case Failure(err)  => println(s"Could not find field 'name': $err")
-      }
-// #end::json4[]
-    }
-
-// #tag::cluster[]
-    val cluster = Cluster.connect("localhost", "username", "password").get
-// #end::cluster[]
+    // tag::cluster[]
+    val cluster = Cluster.connect("localhost", "Administrator", "password").get
+    // end::cluster[]
 
-// #tag::resources[]
-    val bucket = cluster.bucket("bucket-name")
-    val scope = bucket.scope("scope-name")
-    val collection = scope.collection("collection-name")
-// #end::resources[]
+    // tag::resources[]
+    val bucket = cluster.bucket("travel-sample")
+    val scope = bucket.scope("_default")
+    val collection = scope.collection("_default")
+    // end::resources[]
 
-// #tag::apis[]
+    // tag::apis[]
     val asyncApi: AsyncCollection = collection.async
     val reactiveApi: ReactiveCollection = collection.reactive
-// #end::apis[]
+    // end::apis[]
+
+    json1
```

Comment:
Huh, so Scala lets you call functions with no params without parenthesis?
I'd be happy enough with this in Ruby, feels a bit odd in a Javalike.

Reply:
Just reading this because I am not sure what the best practice is here: https://docs.scala-lang.org/style/method-invocation.html

> Scala allows the omission of parentheses on methods of arity-0 (no arguments) 

> However, this syntax should only be used when the method in question has no side-effects (purely-functional). In other words, it would be acceptable to omit parentheses when calling queue.size, but not when calling println(). This convention mirrors the method declaration convention given above.

So maybe I should use parenthesis in this particular case!